### PR TITLE
'Solution'

### PR DESCRIPTION
**Pull request by LaskoA**
Hello, please check my solution :) I have tried to create "print_field" method but faced a problem with allocation of decks of alive ship and decks of drowned ship (line 64 - line 67). Can I have an advice, please? Thank you.

### CODE REVIEW

File: app/main.py [hunk 1]
```diff
@@ -1,34 +1,93 @@
...
-        # If it is, then it should check if this cell is the last alive
-        # in the ship or not.
-        pass
+        if location in self.field:
+            damaged_ship = self.field[location]
+            damaged_ship.fire(location[0], location[1])
+            if damaged_ship.is_drowned:
+                return "Sunk!"
+            return "Hit!"
+        else:
```
Comment (Abnormaltype) [inline app/main.py:92]: remove unnecessary `else`

File: app/main.py [hunk 2]
```diff
@@ -1,34 +1,93 @@
...
+        """
+        battlefield = np.zeros((10, 10), str)
+        rows = battlefield.shape[0]
+        columns = battlefield.shape[1]
+        for i in range(0, rows):
+            for j in range(0, columns):
+                if not self.get_deck(i, j):
+                    battlefield[i, j] = "~"
+                if self.get_deck(i, j):
+                    if Deck(i, j).is_alive:
```
Comment (Abnormaltype) [inline app/main.py:64]: use method `get_deck`

**Review by Abnormaltype — CHANGES_REQUESTED** [review]
also, can write me in Slack

**Comment by LaskoA** [comment]
"print_field" method was relocated from class Ship to class Battleship.

**Review by Abnormaltype — CHANGES_REQUESTED** [review]

**Comment by LaskoA** [comment]
Please check now)

**Review by Abnormaltype — APPROVED** [review]
good job!

**Pull request closed without merging**